### DISCUSSION TITLE
How to retrieve the Video Title?

### DISCUSSION THREAD

**Discussion by Vendrel** [jdepoix/youtube-transcript-api#284] [General]
I see `video_id`, `language_code` but couldn't find a variable for video title.
_'Transcript' object has no attribute 'video_title'_
_'Transcript' object has no attribute 'title'_

**Comment by Vendrel** [comment]
It truly seems there's no attribute for Video Title.

The following code template worked for me:

```
import requests
from bs4 import BeautifulSoup

# YouTube Video ID
video_id = 'YOUR_VIDEO_ID'

# YouTube Video URL
url = f'https://www.youtube.com/watch?v={video_id}'

# Extracting HTML Code of the Video Page:
response = requests.get(url)
html_content = response.text

# Processing the HTML Code with BeautifulSoup
soup = BeautifulSoup(html_content, 'html.parser')

# Extracting <title> tag's content
title_tag = soup.find('meta', property='og:title')
video_title = title_tag['content'] if title_tag else 'Title not found'

print(video_title)

```

**Comment by jdepoix** [comment]
Hi @Vendrel!
That is correct, there is no property for the video title. I generally prefer keeping the scope of this module rather small/concise and therefore try to avoid adding features that require additional requests and parsing of data to add metadata that is not strictly required for retrieving transcripts.

**Reply by tstanescu1** [reply]
Thanks @jdepoix  for the great tool! I'm currently in the process of implementing it in my app, but I have two issues:
1. Same as @Vendrel , I would love to have at least a title when extracting the transcripts, if not I would have to make a second request to another API. I understand that you want it to be lean, but I think a title would be a relatively cheap addition that would bring much benefit, as title and captions go well together. 
2. I understand that the default language is set to "en", but if you try to extract a video, without specifying the language it will throw an error for any language other than "en". Could we possibly make it to be using en OR the default language of the video? This would be super useful as now I have to create 2 requests, one to get the video and check the language, and then make sure to set it.

Very much appreciated Jonas, thanks again!
